models/bsheet_mscoa.json

```diff
@@ -0,0 +1,136 @@
+{
+    "fact_table": "bsheet_facts_mscoa",
```

Comment:
what do you think of naming these something like `bsheet_v2` instead of `bsheet_mscoa` and including something like `Section 71-level aggregation of mSCOA data from 2019-20 onwards replacing the original bsheet cube`

Reply:
Im pondering this since this data isn't actually clasified according to the mSCOA schema, it's classified according to Section 71 version something, based on aggregated mSCOA data. Calling it mSCOA might be confusing and disappointing to experts

Reply:
Makes sense. Making those changes now.

Reply:
Change made @jbothma. Please resolve if you're happy.